### PR TITLE
Adjust/test indication

### PR DESCRIPTION
**Pull request by julianasobreira**
## Contexto
Atualmente uma das principais dúvidas que aparece no time de ativação é sobre transações feitas em test ao invés de live. Não fica claro para os clientes a mudança de ambiente e precisamos enfatizar isso na pilot. 

## Checklist
- [x] Adicionar nova indicação de ambiente de test

## Issues linkadas
[CRED-14](https://mundipagg.atlassian.net/browse/CRED-14)

### Layout:
image

## Como testar?
Logar no ambiente de sandbox.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 9c333822fc35bd5b7700a46fb0419938096ebe57
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 9c333822fc35bd5b7700a46fb0419938096ebe57
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 9c333822fc35bd5b7700a46fb0419938096ebe57
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 9c333822fc35bd5b7700a46fb0419938096ebe57
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 9c333822fc35bd5b7700a46fb0419938096ebe57
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/containers/Header/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (eduardogspereira) [inline packages/pilot/src/containers/Header/index.js]: Esse eslint-disable ficou de leftover? :thinking:

Comment (zignd) [inline packages/pilot/src/containers/Header/index.js:1]: isso aqui era só para teste e ficou para trás?

File: packages/pilot/src/containers/Header/style.css [hunk 2]
```diff
@@ -1,10 +1,38 @@
...
-  color: var(--color-squanchy-gray-400);
-  cursor: pointer;
-  user-select: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 10px 20px 0 20px;
+  padding: 10px 15px;
+  background-color: var(--color-barney-purple-100);
+  color: #ffffff;
```
Comment (eduardogspereira) [inline packages/pilot/src/containers/Header/style.css:10]: É uma boa usar a variavel de cor --color-white aqui?

File: packages/pilot/src/containers/Header/style.css [hunk 3]
```diff
@@ -1,10 +1,38 @@
...
+  border-radius: 6px;
+}
+
+.testEnvironmentLabel p {
+  margin: 0;
+  font-size: 14px;
```
Comment (eduardogspereira) [inline packages/pilot/src/containers/Header/style.css:16]: Melhor passar o font-size e line-height pro `.testEnvironmentLabel` neh? Pq a tag `p` e `a` irá herdar essas propriedades

File: packages/pilot/src/containers/Header/index.js [hunk 4]
```diff
@@ -37,7 +38,19 @@ const getEnvironmentUrl = () => (
     : testUrl
 )
 
-const renderEnvironmentButton = ({
+const renderTestEnviromentNav = () => (
+  <div className={style.testEnvironmentLabel}>
+    <p>Você está em ambiente de testes, operações realizadas neste ambiente
```
Comment (eduardogspereira) [inline packages/pilot/src/containers/Header/index.js:43]: Não é melhor esse texto estar no translations :thinking:

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: af4f9062d3d8fa711975c02e2e01f1124b633bae
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: af4f9062d3d8fa711975c02e2e01f1124b633bae
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: af4f9062d3d8fa711975c02e2e01f1124b633bae
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: af4f9062d3d8fa711975c02e2e01f1124b633bae
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Pull request merged**